tap_zendesk_chat/streams.py

```diff
@@ -145,7 +145,7 @@ def _pull(self, ctx, chat_type, ts_field, full_sync, schema: Dict, stream_metada
                 if next_url:
                     search_resp = ctx.client.request(self.tap_stream_id, url=next_url)
                 else:
-                    params = {"q": f"type:{chat_type} AND {ts_field}:[{start_dt.isoformat()} TO {end_dt.isoformat()}]"}
+                    params = {"q": f"type:{chat_type} AND {ts_field}:[{start_dt.replace(tzinfo=None).isoformat()} TO {end_dt.replace(tzinfo=None).isoformat()}]"}
```

Comment:
Is this the right place [utils.py#L20](https://github.com/singer-io/tap-zendesk-chat/blob/311e21553ff84067814fc461d3da896a31ea8e41/tap_zendesk_chat/utils.py#L20) to fix this issue?

Reply:
Existing implementation is simpler than the suggested one. So, going with the current approach.